packages/config/config/devices/0x001d/dz6hd.json

```
@@ -50,30 +62,78 @@
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 100,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Memory Dim",
```

```suggestion
					"label": "Last non-zero level",
```
Is probably a bit clearer (if I understand this correctly?)